src/db/scripts/initAll.js

```diff
@@ -81,27 +80,31 @@ export const initMainPageState = async metadataTestingUser => {
   const tags = await Tag.find().exec();
   const tagsBySlug = keyBy(tags, 'slug');
   const articlesByTag = getArticlesByTag({ articles, tags });
-  const { personalities = [], locations = [] } = getTagsByTopic({ tags, topics });
-
-  // TODO: filter out personalities with 0 articles (diary authors)
```

Comment:
now we're doing it both on Backend and Frontend ?

Reply:
in different places